docs/preview/features/secret-store/provider/environment-variables.md

```diff
@@ -33,6 +33,9 @@ public class Program
 
                        // Uses the environment variables starting with 'ARCUS_' from the environment block associated with the current process.
                        builder.AddEnvironmentVariables(prefix: "ARCUS_");
+
+                       // Uses the environment variables, using underscores and capitals for secret name structure.
+                       builder.AddEnvironmentVariables(mutateSecretName: name => name.Replace(".", "_").ToUpper());
```

Comment:
Good example! Maybe even extend it to include a prefix:
```suggestion
                       // Example - When looking up Queue.Name it will be changed to ARCUS_QUEUE_NAME
                       builder.AddEnvironmentVariables(mutateSecretName: name => $"ARCUS_{name.Replace(".", "_").ToUpper()}");
```